packages/intrepid2/src/Shared/Intrepid2_Data.hpp

```diff
@@ -1352,6 +1767,25 @@ namespace Intrepid2 {
       return false; // statement should be unreachable; included because compilers don't necessarily recognize that fact...
     }
     
+    //! Constructs a container suitable for storing the result of an in-place combination of the two provided data containers.  The two containers must have the same nominal shape.
```

Comment:
nominal shape -> logical shape?

Reply:
nevermind.

Reply:
@mperego, yes, logical would be better here.  If that turns out to be the only change, I'll make it in the next PR (I've got a smaller one that I'd like to do after this one is in), just to avoid having to rerun the tests to correct a comment.

Reply:
(Actually, now I see that I have quite a few mentions of "nominal" still in the comments.  Again, I can take of that in the next PR if there aren't substantive changes you'd like in this one.)

Reply:
Sure, that's fine.